src/Tizen.NUI/src/public/BaseComponents/AnimatedVectorImageView.cs

```diff
@@ -596,6 +596,25 @@ public void SetMinAndMaxFrameByMarker(string marker1, string marker2 = null)
             NUILog.Debug($" [{GetId()}] ]AnimatedVectorImageView END]");
         }
 
+        /// <summary>
+        /// Resume Animation.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new void Resume()
```

Comment:
Why are you using the 'new' keyword here? I see it used quite frequently throughout the implementation of existing classes, including this function. Is there any specific reason behind this choice?

```suggestion
        public void Resume()
```